api_tests/collections/test_views.py

```diff
@@ -3204,6 +3204,29 @@ def test_delete_not_present(
         assert len(res.json['data']) + \
             len(reg_res.json['data']) == number_of_links
 
+    def test_delete_linked_registration(
+            self, app, make_payload, url_private_linked_regs, user_two,
+            collection_private, user_one, registration_private):
+
+        payload = {
+            'data': [
+                {'id': registration_private._id, 'type': 'linked_registrations'}
```

Comment:
Given #8917, should we test with `'type': 'registrations'` (version >= 2.13) as well?

Reply:
ah, thanks, I missed that this had gone in.

It doesn't look like `type` is enforced though for DELETE requests - we're normally enforcing type checking on the serializer, but deleting is handled on the view [CollectionLinkedNodesRelationship.perform_destroy](https://github.com/CenterForOpenScience/osf.io/blob/develop/api/collections/views.py#L816).  We just loop through the id's in data and remove them.  I could enforce type on this view, I just don't know if this is the proper place to put it/do we need to be strict about DELETE's?

Reply:
Strictly speaking I'd expect a 400 if I tried to delete a `type: 'donut'`, and that sort of logic should be implemented only once (not sure where)... but I think adding that is pretty low priority. I'm content with just the test 👍